docs/scripts/docs.js

```diff
@@ -0,0 +1,24 @@
+// eslint-disable-next-line no-unused-vars
+function tryInREPL(event, scope) {
```

Comment:
The `scope` can be safely deleted, so do the eslint comment.

Reply:
`scope` is used and if we delete the comment then eslint will complain about `tryInREPL` is not used

Reply:
Ok, thanks for the info.